sdk/purview/purview-sharing-rest/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2023-06-23)
```

Comment:
Does that mean the release date is finalized?

Reply:
yes for the forseaable future.

Reply:
No, the release date is up to you and I just confirm that.